src/Tools/ExternalAccess/FSharp/Internal/Editor/FSharpEditorFormattingService.cs

```diff
@@ -17,6 +18,7 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Internal.Editor
 {
     [Shared]
     [ExportLanguageService(typeof(IEditorFormattingService), LanguageNames.FSharp)]
+    [Obsolete($"Move to  {nameof(IFSharpFormattingInteractionService)} now")]
```

Comment:
❓ Why moving? The external access layer is supposed to allow the same type to continue working.

Reply:
The intent is to move them to the new interface with the new return types. If that's not a goal, I can certainly adjust. But that's the intent.

Reply:
➡️ F# team can make the call on if/when to move to the new service. External access layer exists for the purpose of isolating our decision making from theirs. For this PR, we act as though there is no intent to change (with essentially no downside).